tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Simple RAID partitioning layout validation for encrypted home
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use Mojo::JSON qw(decode_json);
+
+sub run {
+    select_console 'root-console';
+
+    # To avoid the regression of bsc#1224393, need ensure that the crypt /home
+    # partition is under 'md1' via comparing with the partitioning output of md1.
+    my $lsblk_output = decode_json(script_output("lsblk -M -J /dev/md1"));
+    my $dev_name = $lsblk_output->{blockdevices}->[0]->{name};
```

Comment:
iirc, this syntax could be used to shorten chained references:
`my $dev_name = $lsblk_output->{blockdevices}[0]{name};`

Reply:
Updated.